### ISSUE TITLE
docker-compose:  services.kai.environment.DEMO_MODE contains false, which is an invalid type, it should be a string, number, or a null

### ISSUE THREAD

**Issue by jwmatthews** [konveyor-ecosystem/kai#227]
I am running on an older Mac (x86_64), I have docker and podman installed, yet I am only using podman.
When I run `podman compose up`, I'm seeing an error from `docker-compose` being invoked.
Looking at `man podman-compose` the use of `docker-compose` seems normal and expected.

Error is below, looks like we need to tweak syntax of compose.yaml so it can work with both docker-compose and podman-compose.

```
$ time podman compose up
>>>> Executing external compose provider "/usr/local/bin/docker-compose". Please refer to the documentation for details. <<<<

WARNING: The POSTGRES_USER variable is not set. Defaulting to a blank string.
ERROR: The Compose file './compose.yaml' is invalid because:
services.kai.environment.DEMO_MODE contains false, which is an invalid type, it should be a string, number, or a null
Error: executing /usr/local/bin/docker-compose up: exit status 1
podman compose up  0.39s user 0.17s system 44% cpu 1.253 total

$ which podman
/usr/local/bin/podman

[jmatthews] in ~/git/jwmatthews/kai (main) $ podman compose version
>>>> Executing external compose provider "/usr/local/bin/docker-compose". Please refer to the documentation for details. <<<<

docker-compose version 1.29.2, build 5becea4c
docker-py version: 5.0.0
CPython version: 3.9.0
OpenSSL version: OpenSSL 1.1.1h  22 Sep 2020

$ which docker-compose
/usr/local/bin/docker-compose

$ docker-compose version
docker-compose version 1.29.2, build 5becea4c
docker-py version: 5.0.0
CPython version: 3.9.0
OpenSSL version: OpenSSL 1.1.1h  22 Sep 2020

```

**Comment by jwmatthews** [comment]
Below change is sufficient for working with `docker-compose`, will test on podman-compose and submit a fix shortly

```
$ git diff
diff --git a/compose.yaml b/compose.yaml
index b66fa13..a4fab7c 100644
--- a/compose.yaml
+++ b/compose.yaml
@@ -10,7 +10,7 @@ services:
     environment:
       <<: *common-variables
       LOGLEVEL: info
-      DEMO_MODE: False
+      DEMO_MODE: "False"
       NUM_WORKERS: 8
       # Do not edit the variables below otherwise
       # you risk committing keys to a public repo

```

**Comment by jwmatthews** [comment]
Fixed in #225
